### PR TITLE
[hi] Update Hindi readme file

### PR DESCRIPTION
**Pull request by haripriya9647**
#41989

### CODE REVIEW

**Comment by haripriya9647** [comment]
@divya-mohan0209 requesting your suggestions

File: README-hi.md [hunk 1]
```diff
@@ -7,11 +7,11 @@
 
 ## डॉक्स में योगदान देना
 
-आप अपने GitHub खाते में इस रिपॉजिटरी की एक copy बनाने के लिए स्क्रीन के ऊपरी-दाएँ क्षेत्र में **Fork** बटन पर क्लिक करें। इस copy को *Fork* कहा जाता है। अपने fork में परिवर्तन करने के बाद जब आप उनको हमारे पास भेजने के लिए तैयार हों, तो अपने fork पर जाएँ और हमें इसके बारे में बताने के लिए एक नया pull request बनाएं।
+अगर आप इस रिपॉजिटरी की एक अपनी copy बनाना चाहते हैं, तो स्क्रीन के ऊपरी-दाएँ क्षेत्र में **Fork** बटन पर क्लिक करें। इसको हम Fork कहते हैं, जो आपकी खुद की एक नई copy होती है। जब आप इस fork में कुछ बदलाव करके हमें भेजने के लिए तैयार हो जाएं, तो अपने fork पर जाएं और एक नया pull request बनाएं, ताकि हमें आपके किए गए परिवर्तन के बारे में पता चल सके।
```
Comment (aj11anuj) [inline README-hi.md:10]:
```suggestion
अगर आप इस रिपॉजिटरी की एक copy बनाना चाहते हैं, तो स्क्रीन के ऊपरी-दाएँ क्षेत्र में **Fork** बटन पर क्लिक करें। इसको हम Fork कहते हैं, जो आपकी खुद की एक नई copy होती है। जब आप इस fork में कुछ बदलाव करके हमें भेजने के लिए तैयार हो जाएं, तो अपने fork पर जाएं और एक नया pull request बनाएं, ताकि हमें आपके किए गए परिवर्तन के बारे में पता चल सके।
```

File: README-hi.md [hunk 2]
```diff
@@ -7,11 +7,11 @@
...
 
-एक बार जब आपका pull request बन जाता है, तो एक कुबरनेट्स समीक्षक स्पष्ट, कार्रवाई योग्य प्रतिक्रिया प्रदान करने की जिम्मेदारी लेगा। pull request के मालिक के रूप में, **यह आपकी जिम्मेदारी है कि आप कुबरनेट्स समीक्षक द्वारा प्रदान की गई प्रतिक्रिया को संबोधित करने के लिए अपने pull request को संशोधित करें।**
+एक बार जब आपका pull request तैयार हो जाता है, तो हमारे कुबरनेट्स समीक्षक आपके सुझाव और प्रतिक्रिया को ध्यान से देखेंगे। आपकी pull request की मालिकी के रूप में, **यह आपकी जिम्मेदारी है कि आप हमारे समीक्षकों द्वारा दी गई प्रतिक्रिया का उचित रूप से संबोधित करने के लिए अपने पुल रिक्वेस्ट को संशोधित करें। हम साथ में काम करेंगे ताकि हमारा कोड सही और बेहतर हो सके।**
```
Comment (aj11anuj) [inline README-hi.md:12]:
```suggestion
एक बार जब आपका pull request तैयार हो जाता है, तो हमारे कुबरनेट्स समीक्षक आपके सुझाव और प्रतिक्रिया को ध्यान से देखेंगे। आपकी pull request की मालिकी के रूप में, **यह आपकी जिम्मेदारी है कि आप हमारे समीक्षकों द्वारा दी गई प्रतिक्रिया का उचित रूप से संबोधित करने के लिए अपनी पुल रिक्वेस्ट को संशोधित करेंगे। हम साथ में काम करेंगे ताकि हमारा कोड सही और बेहतर हो सके।**
```

File: README-hi.md [hunk 3]
```diff
@@ -7,11 +7,11 @@
...
 
-यह भी ध्यान दें कि आप एक से अधिक कुबरनेट्स समीक्षक आपको प्रतिक्रिया प्रदान कर सकते हैं या आप एक कुबरनेट्स समीक्षक से प्रतिक्रिया प्राप्त कर सकते हैं जो मूल रूप से आपको प्रतिक्रिया प्रदान करने के लिए दिए गए एक से भिन्न है। इसके अलावा, कुछ मामलों में, आपका एक समीक्षक जरूरत पड़ने पर [कुबेरनेट्स टेक समीक्षक](https://github.com/kubernetes/website/wiki/Tech-reviewers) से तकनीकी समीक्षा प्राप्त कर सकता है। समीक्षक समय पर प्रतिक्रिया देने के लिए पूरी कोशिश करेंगे, लेकिन परिस्थितियों के आधार पर प्रतिक्रिया समय अलग-अलग हो सकता है।
+यह भी ध्यान दें कि एक से अधिक कुबरनेट्स समीक्षक आपको प्रतिक्रिया देने के लिए आ सकते हैं या फिर आप एक समीक्षक से प्रतिक्रिया प्राप्त कर सकते हैं, जो मूल रूप से आपके पुल रिक्वेस्ट की जांच कर रहे हैं। इसके अलावा, कुछ मामलों में, आपको आपके प्रोजेक्ट के लिए [कुबेरनेट्स टेक समीक्षक](https://github.com/kubernetes/website/wiki/Tech-reviewers) से तकनीकी समीक्षा प्राप्त करने की आवश्यकता हो सकती है। समीक्षक समय पर प्रतिक्रिया देने के लिए पूरी कोशिश करेंगे, लेकिन हम समझते हैं कि सभी के पास व्यस्तता होती है और परिस्थितियों के आधार पर प्रतिक्रिया में विलंब हो सकता है।
```
Comment (aj11anuj) [inline README-hi.md:14]:
```suggestion
यह भी ध्यान दें कि एक से अधिक कुबरनेट्स समीक्षक आपको प्रतिक्रिया देने के लिए आ सकते हैं, जो मूल रूप से आपकी पुल रिक्वेस्ट की जांच करेंगे। इसके अलावा, कुछ मामलों में, आपको आपके प्रोजेक्ट के लिए [कुबेरनेट्स टेक्निकल समीक्षक](https://github.com/kubernetes/website/wiki/Tech-reviewers) से तकनीकी समीक्षा प्राप्त करने की आवश्यकता हो सकती है। समीक्षक समय पर प्रतिक्रिया देने की पूरी कोशिश करेंगे, लेकिन हम समझते हैं कि सभी के पास व्यस्तता होती है और परिस्थितियों के आधार पर प्रतिक्रिया में विलंब हो सकता है।
```

File: README-hi.md [hunk 4]
```diff
@@ -33,13 +33,13 @@
 
 ## स्थानीय रूप से डॉकर का उपयोग करके साइट चलाना
```
Comment (aj11anuj) [inline README-hi.md:34]:
```suggestion
## स्थानीय रूप से डॉकर का उपयोग करके साइट चलायें
```

File: README-hi.md [hunk 5]
```diff
@@ -80,6 +80,7 @@ make serve
 
 ## धन्यवाद!
 
-कुबरनेट्स सामुदायिक भागीदारी पर पनपती है, और हम वास्तव में हमारी साइट और हमारे प्रलेखन में आपके योगदान की सराहना करते हैं!
 
-कुबरनेट्स आपकी भागीदारी पर निर्भर है, और हम हमारी साइट और प्रलेखन में आपके योगदान का मान करते हैं!
+कुबरनेट्स सामुदायिक भागीदारी में बढ़ता है, और हम सचमुच हमारी साइट और प्रलेखन में आपके योगदान की सराहना करते हैं!
+
+कुबरनेट्स आपकी भागीदारी की कड़ी से जुड़ा हुआ है, और हम हमेशा आपके सहयोग के लिए आभारी हैं।
```
Comment (aj11anuj) [inline README-hi.md:86]:
```suggestion
कुबरनेट्स सामुदायिक भागीदारी से बढ़ता है, और हम सचमुच हमारी साइट और प्रलेखन में आपके योगदान की सराहना करते हैं!

कुबरनेट्स आपकी भागीदारी की कड़ी से जुड़ा हुआ है, और हम हमेशा आपके सहयोग के लिए आभारी रहेंगे।
```

File: README-hi.md [hunk 6]
```diff
@@ -33,13 +33,13 @@
 
 ## स्थानीय रूप से डॉकर का उपयोग करके साइट चलाना
 
-कुबरनेट्स वेबसाइट को स्थानीय रूप से चलाने के लिए अनुशंसित तरीका एक विशेष [डॉकर](https://docker.com) image को चलाना है, जिसमें [Hugo](https://gohugo.io) स्टेटिक साइट जनरेटर शामिल है।
+कुबरनेट्स वेबसाइट को स्थानीय रूप से चलाने के लिए, हम आपको एक विशेष [डॉकर](https://docker.com) image का इस्तेमाल करने की सिफारिश करते हैं, जिसमें [Hugo](https://gohugo.io) स्टेटिक साइट जनरेटर शामिल है।
```
Comment (aj11anuj) [inline README-hi.md:36]:
```suggestion
कुबरनेट्स वेबसाइट को स्थानीय रूप से चलाने के लिए, हम आप से एक विशेष [डॉकर](https://docker.com) image को इस्तेमाल करने की सिफारिश करते हैं, जिसमें [Hugo](https://gohugo.io) स्टेटिक साइट जनरेटर शामिल है।
```

File: README-hi.md [hunk 7]
```diff
@@ -33,13 +33,13 @@
...
 
-> यदि आप विंडोज पर हैं, तो आपको कुछ और टूल्स की आवश्यकता होगी जिन्हें आप [Chocolatey](https://chocolatey.org) के साथ इंस्टॉल कर सकते हैं।
+> अगर आप विंडोज पर हैं, तो आपको कुछ और उपकरणों की आवश्यकता है, जिन्हें आप [Chocolatey](https://chocolatey.org) का इस्तेमाल करके आसानी से इंस्टॉल कर सकते हैं। यह आपको चलने के लिए एक सरल प्रक्रिया प्रदान करेगा।
 
-यदि आप डॉकर के बिना स्थानीय रूप से वेबसाइट चलाना पसंद करते हैं, तो नीचे दिए गए Hugo का उपयोग करके स्थानीय रूप से [साइट को चलाने](#hugo-का-उपयोग-करते-हुए-स्थानीय-रूप-से-साइट-चलाना) का तरीका देखें।
+अगर आप डॉकर का इस्तेमाल किए बिना ही स्थानीय रूप से वेबसाइट चलाना पसंद करते हैं, तो आप नीचे दिए गए Hugo का उपयोग करके स्थानीय रूप से [साइट को चलाने](#hugo-का-उपयोग-करते-हुए-स्थानीय-रूप-से-साइट-चलाना) का तरीका देख सकते हैं। यह आपको एक सुगम अनुभव प्रदान करेगा जिसमें डॉकर की जरूरत नहीं होगी।
 
-यदि आप [डॉकर](https://www.docker.com/get-started) चला रहे हैं, तो स्थानीय रूप से `कुबेरनेट्स-ह्यूगो` Docker image बनाएँ:
+अगर आप [डॉकर](https://www.docker.com/get-started) का इस्तेमाल कर रहे हैं, तो स्थानीय रूप से `कुबेरनेट्स-ह्यूगो` Docker image बनाने की कदमों को देखें:
```
Comment (aj11anuj) [inline README-hi.md:42]:
```suggestion
अगर आप [डॉकर](https://www.docker.com/get-started) का इस्तेमाल कर रहे हैं, तो स्थानीय रूप से `Kubernetes-Hugo` Docker image बनाने की प्रक्रिया को देखें:
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some minor suggestions from my side

**Comment by sftim** [comment]
Also see https://github.com/kubernetes/website/pull/45657

**Comment by divya-mohan0209** [comment]
@haripriya9647 : Please could you advise if you have the bandwidth to address the reviews suggested by Anuj?

**Comment by haripriya9647** [comment]
yes @divya-mohan0209 I will add commits by tonight

**Comment by sftim** [comment]
Awaiting update from Hindi team (@haripriya9647)

**Comment by niranjandarshann** [comment]
@haripriya9647 The changes looks good to me but  there is lack of  some consistency of using the Word `pull request` instead of using `पुल रिक्वेस्ट` . SomeWhere you use as `पुल रिक्वेस्ट` many where you used  `pull request`.

So use  `pull request ` in entire page.

File: README-hi.md [hunk 8]
```diff
@@ -7,11 +7,11 @@
 
 ## डॉक्स में योगदान देना
 
-आप अपने GitHub खाते में इस रिपॉजिटरी की एक copy बनाने के लिए स्क्रीन के ऊपरी-दाएँ क्षेत्र में **Fork** बटन पर क्लिक करें। इस copy को *Fork* कहा जाता है। अपने fork में परिवर्तन करने के बाद जब आप उनको हमारे पास भेजने के लिए तैयार हों, तो अपने fork पर जाएँ और हमें इसके बारे में बताने के लिए एक नया pull request बनाएं।
+अगर आप इस रिपॉजिटरी की एक copy बनाना चाहते हैं, तो स्क्रीन के ऊपरी-दाएँ क्षेत्र में **Fork** बटन पर क्लिक करें। इसको हम Fork कहते हैं, जो आपकी खुद की एक नई copy होती है। जब आप इस fork में कुछ बदलाव करके हमें भेजने के लिए तैयार हो जाएं, तो अपने fork पर जाएं और एक नया pull request बनाएं, ताकि हमें आपके किए गए परिवर्तन के बारे में पता चल सके।
```
Comment (divya-mohan0209) [inline README-hi.md:10]: Either pull request or the transliterated version, please.
```suggestion
अगर आप इस रिपॉजिटरी की एक copy बनाना चाहते हैं, तो स्क्रीन के ऊपरी-दाएँ क्षेत्र में **Fork** बटन पर क्लिक करें। इसको हम Fork कहते हैं, जो आपकी खुद की एक नई copy होती है। जब आप इस fork में कुछ बदलाव करके हमें भेजने के लिए तैयार है, तो अपने fork पर जाएं और एक नया pull request बनाएं, ताकि हमें आपके किए गए परिवर्तन के बारे में पता चल सके।
```

File: README-hi.md [hunk 9]
```diff
@@ -7,11 +7,11 @@
...
 
-यह भी ध्यान दें कि आप एक से अधिक कुबरनेट्स समीक्षक आपको प्रतिक्रिया प्रदान कर सकते हैं या आप एक कुबरनेट्स समीक्षक से प्रतिक्रिया प्राप्त कर सकते हैं जो मूल रूप से आपको प्रतिक्रिया प्रदान करने के लिए दिए गए एक से भिन्न है। इसके अलावा, कुछ मामलों में, आपका एक समीक्षक जरूरत पड़ने पर [कुबेरनेट्स टेक समीक्षक](https://github.com/kubernetes/website/wiki/Tech-reviewers) से तकनीकी समीक्षा प्राप्त कर सकता है। समीक्षक समय पर प्रतिक्रिया देने के लिए पूरी कोशिश करेंगे, लेकिन परिस्थितियों के आधार पर प्रतिक्रिया समय अलग-अलग हो सकता है।
+यह भी ध्यान दें कि एक से अधिक कुबरनेट्स समीक्षक आपको प्रतिक्रिया देने के लिए आ सकते हैं, जो मूल रूप से आपकी पुल रिक्वेस्ट की जांच करेंगे। इसके अलावा, कुछ मामलों में, आपको आपके प्रोजेक्ट के लिए [कुबेरनेट्स टेक्निकल समीक्षक](https://github.com/kubernetes/website/wiki/Tech-reviewers) से तकनीकी समीक्षा प्राप्त करने की आवश्यकता हो सकती है। समीक्षक समय पर प्रतिक्रिया देने की पूरी कोशिश करेंगे, लेकिन हम समझते हैं कि सभी के पास व्यस्तता होती है और परिस्थितियों के आधार पर प्रतिक्रिया में विलंब हो सकता है।
```
Comment (divya-mohan0209) [inline README-hi.md:14]:
```suggestion
यह भी ध्यान दें कि एक से अधिक कुबरनेट्स समीक्षक मूल रूप से आपकी पुल रिक्वेस्ट की जांच कर सकते हैं और आपको प्रतिक्रिया दे सकते हैं। इसके अलावा, कुछ मामलों में, आपको आपके प्रोजेक्ट के लिए [कुबेरनेट्स टेक्निकल समीक्षक](https://github.com/kubernetes/website/wiki/Tech-reviewers) से तकनीकी समीक्षा प्राप्त करने की आवश्यकता हो सकती है। समीक्षक समय पर प्रतिक्रिया देने की पूरी कोशिश करेंगे, लेकिन हम समझते हैं कि सभी के पास व्यस्तता होती है और परिस्थितियों के आधार पर प्रतिक्रिया में विलंब हो सकता है।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
@haripriya9647 - The contents of [this file](https://github.com/kubernetes/website/blob/main/README.md) and the translated version don't match. This seems to be outdated & you will need to update the Hindi README to match the English one.

**Pull request closed without merging**